pkg/terminal/colorize/colorize.go

```diff
@@ -92,6 +90,15 @@ func Print(out io.Writer, path string, reader io.Reader, startLine, endLine, arr
 		}
 
 		switch n := n.(type) {
+		case *ast.File:
+			emit(token.PACKAGE, f.Package, token.NoPos)
+			return true
+		case *ast.FuncType:
```

Comment:
Remove this case and change the code:

```
				kwpos, ok := kwposval.Interface().(token.Pos)
				if ok {
					emit(token.ILLEGAL, kwpos, token.NoPos)
}
```

to say `if ok && kwpos != token.NoPos`

Reply:
case *ast.FuncType is removed, and the code mentioned above is changed.

done